components/tabstrip/overview.md

```diff
@@ -13,18 +13,18 @@ position: 0
 This article provides information about the <a href = "https://www.telerik.com/blazor-ui/tabstrip" target = "_blank">Blazor TabStrip component</a> and its core features.
 
 >caption In this article:
-* [Basics](#basics)
+* [Creating Blazor TabStrip](#creating-blazor-tabstrip)
 * [Features](#features)
```

Comment:
We should list the features themselves and remove the Features heading. See /wiki/Docs-Overview-Page-Template in the private repo.